app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -68,8 +70,40 @@ html.omni-nav-v2 {
   table.gsc-search-box {
     margin-bottom: 0;
 
+    tr { line-height: 25px; }
+
     td.gsc-input {
        padding-right: 0;
+
+      div.gsc-input-box-focus,
+      div.gsc-input-box-hover, { box-shadow: none; }
+
+      div.gsc-input-box {
+        border: none;
+        height: 28px;        
+      }
+    }
+
+    .gsib_a {
+      padding: 1px 0px 1px 6px;
+
+      input.gsc-input {
+        font-size: 13px;
+      }
+    }
+
+    .gsib_b { display: none; }
+
+    .gsc-clear-button {
+      display: table-cell;
+      padding-left: 5px;
+      div.gsc-clear-button { background-image: url('//www.google.com/uds/css/clear.gif'); }
+    }
+
+    //fixes issue with WAVE accessibility errors for empty button element google search added to search box
+    span {
```

Comment:
Move this block under the `button.gsc-search-button, input.gsc-search-button` selector on line 128